quickwit/quickwit-indexing/src/source/ingest_api_source.rs

```diff
@@ -145,18 +145,15 @@ impl Source for IngestApiSource {
         };
 
         // TODO use a timestamp (in the raw doc batch) given by at ingest time to be more accurate.
-        let mut raw_doc_batch = RawDocBatch::default();
-        raw_doc_batch.docs.extend(
-            doc_batch
-                .iter()
-                .map(|command| match command {
-                    DocCommand::Ingest { payload } => payload,
-                    // TODO: We will add a commit logic in a follow-up PR, for now - just panic
-                    _ => panic!("Unknown command {command:?}"),
-                })
-                .map(|buff| String::from_utf8_lossy(buff.as_ref()).to_string()),
```

Comment:
This line illustrates the purpose of this PR.

Reply:
Makes sense.